tests/src/test/scala-3.x/cats/tests/FunctionKLiftSuite.scala

```diff
@@ -0,0 +1,16 @@
+package cats.tests
+
+import cats.arrow.FunctionK
+import cats.implicits._
+import org.scalacheck.Prop._
+import cats.laws.discipline.arbitrary._
+
+class FunctionKLiftSuite extends CatsSuite {
+
+  test("lift a polymorphic function directly") {
+    val fHeadOption = FunctionK.lift[List, Option]([X] => (_: List[X]).headOption)
```

Comment:
Note that in the future the compiler should support polymorphic sam, so you'll be able to write `val fHeadOption: List ~> Option = [X] => (_: List[X]).headOption` (or maybe even `= _.headOption`)